### DISCUSSION TITLE
Color Theme for Spec files

### DISCUSSION THREAD

**Discussion by nagamanickamm** [getgauge/gauge#2257] [Ideas]
Is there any color theme available for guage spec files in VS code?

Currently except the scenario and feature name other texts just display in White color.
The whole thing looks like some notes written on Notepad rather than looking like a test document.

It would be good to highlight
* Tags
* Tables
* Scenarios
* Text within Dynamic parameter Keywords such as <> ""

**Comment by Haratsu** [comment]
I also tried the anti-pattern of embedding plain old html into my specs which is ok if shown in a markdown viewer, but fails when using `gauge docs spectacle` command as this treats all html tags as plain text (probably for security reaons).
